packages/taco/test/conditions/compound-condition.test.ts

```diff
@@ -185,9 +196,9 @@ describe('validation', () => {
   ])('accepts shorthand for "%s" operator', (operator, operands, expected) => {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const compoundConditionShorthands: Record<string, any> = {
-      'and': CompoundCondition.and,
-      'or': CompoundCondition.or,
-      'not': CompoundCondition.not,
+      and: CompoundCondition.and,
+      or: CompoundCondition.or,
+      not: CompoundCondition.not,
```

Comment:
How does removing the quotes work here?

Reply:
Huh...I still don't get it. How would it work if your key were the string `"foo: bar"`?
(In any case, it's not important for this PR, just wanting to understand more. TS/JS can be weird.)

Reply:
These are equivalent
```
const a = { my_key: 1 };
const b = { "my_key": 1 };
// a == b
```

This however, will not compile:
```
const a = { foo: bar: 1 }; // whoops, throws compilation error
const b = { "foo: bar": 1 }; // works, amazingly
```

We can use `"string notation"` to use actual strings as object keys. This works similarly how you can declare a dict in Python.

The second notation is the object notation. The keys are actually object properties so they have to follow the syntax rules for objects.